include/msckd.h

```diff
@@ -54,8 +54,10 @@
 #ifndef MSCKD_H_
 #define MSCKD_H_
 
+#include "opencv2/core/core.hpp"
```

Comment:
The advantage is when openfabmap is being compiled by the user as a standalone project (outside of OpenCV). If it includes `opencv.hpp`, that will include the OpenCV-internal version of openfabmap. Maybe this isn't the case now with opencv-contrib, but it seemed to be the cleanest. If you're not planning on maintaining an OpenCV external version, then it might not matter.